docs/pages/recipes/multi-chain-setup.mdx

```diff
@@ -10,6 +10,61 @@ Smart Wallets supports multi-chain apps, allowing you to build applications that
 
 In order to support multiple chains in your app, the first thing you need to do is update your `createConfig` call to include the chains you want to support.
 
+<Tabs>
+  <Tab title="React">
+
+```ts twoslash config.ts
+// @noErrors
+import { createConfig } from "@account-kit/react";
+import { sepolia, mainnet } from "@account-kit/infra";
+
+export const config = createConfig({
+  apiKey: "ALCHEMY_API_KEY",
+  // this is the default chain
+  chain: sepolia,
+  chains: [
+    {
+      chain: mainnet, // optional: you can specify a policy ID for this chain, if you want to sponsor gas
```

Comment:
Nit: should this comment be on a new line?

Reply:
hmm yup probably, I just relocated this file. I'll move this comment down